pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -226,6 +228,89 @@ var _ = Describe("podNIC", func() {
 			storedState: ptr(cache.PodIfaceNetworkingPrepared),
 		}),
 	)
+	Context("state transition", func() {
+		When("state is PodIfaceNetworkingUnprepared", func() {
+			var (
+				podnic *podNIC
+			)
+			BeforeEach(func() {
+				var err error
+				vmi := newVMIBridgeInterface("testnamespace", "testVmName")
+				podnic, err = newPhase1PodNICWithMocks(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				pid := 1
+				podnic.launcherPID = &pid
+				podnic.setState(cache.PodIfaceNetworkingUnprepared)
+				address1 := &net.IPNet{IP: net.IPv4(1, 2, 3, 4)}
+				address2 := &net.IPNet{IP: net.IPv4(169, 254, 0, 0)}
+				fakeAddr1 := netlink.Addr{IPNet: address1}
+				fakeAddr2 := netlink.Addr{IPNet: address2}
+				mockNetwork.EXPECT().ReadIPAddressesFromLink(primaryPodInterfaceName).Return(fakeAddr1.IP.String(), fakeAddr2.IP.String(), nil)
+				mockNetwork.EXPECT().IsIpv4Primary().Return(true, nil).Times(1)
+				mockPodNetworkConfigurator.EXPECT().DiscoverPodNetworkInterface(podnic.podInterfaceName).Times(1)
+				mockPodNetworkConfigurator.EXPECT().GenerateDomainIfaceSpec().Return(api.Interface{})
+				mockPodNetworkConfigurator.EXPECT().GenerateNonRecoverableDHCPConfig().Return(&cache.DHCPConfig{}).Times(1)
```

Comment:
`Times(1)` is redundant:
https://github.com/golang/mock/blob/b96f1750f0589c378bad084ee97314192868b2d0/gomock/call.go#L79